sdks/python/apache_beam/runners/common.py

```diff
@@ -477,8 +482,24 @@ def __init__(self,
 
     # Optimize for the common case.
     main_receivers = tagged_receivers[None]
+
+    # TODO(BEAM-3937): Remove if block after output counter released.
+    experiments = RuntimeValueProvider.get_value('experiments', str, [])
+    per_element_output_counter = None
```

Comment:
Nit: I would put this in an else clause.

Reply:
Fixed in latest commit!